src/services/gameleaderboard.service.ts

```diff
@@ -183,7 +183,7 @@ export class GameLeaderboardService {
     const [userOneEntry] = entries.filter((e) => e.userId == userOne.id);
 
     if (!userOneEntry) {
-      return 'User not found';
+      return `User with id ${userOne.id} not found`;
```

Comment:
prob should return their nickname/username
The user wont know what the id means since its just a bunch of random numbers